USAGE.md

```diff
@@ -134,46 +137,42 @@ pandoc:
+#### Citation aliases
 
-The system also supports citation tags, which map from one citation key (an alias) to another.
+The system also supports citation aliases, which map from one citation key (the "alias" or "tag") to another.
 Tags are recommended for the following applications:
 
-1. A citation's identifier contains forbidden characters, you must use a tag.
+1. A citation key contains forbidden characters.
 2. A single reference is cited many times.
-   Therefore, it might make sense to define a tag, so if the citation updates (e.g. a newer version becomes available), only a single change is required.
+   Therefore, it might make sense to define an alias, so if the citation updates (e.g. a newer version becomes available), only a single change is required.
 
-Tags can be defined using Markdown's [link reference syntax](https://spec.commonmark.org/0.29/#link-reference-definitions) as follows:
+Aliases can be defined using Markdown's [link reference syntax](https://spec.commonmark.org/0.29/#link-reference-definitions) as follows:
 
 ```markdown
-Citing a URL containing a `?` character [@tag:my-url].
-Citing a DOI containing parentheses [@doi:my-doi].
+Citing a URL containing a `?` character [@my-url].
+Citing a DOI containing parentheses [@my-doi].
 
-[@tag:my-url]: url:https://openreview.net/forum?id=HkwoSDPgg
-[@tag:my-doi]: doi:10.1016/S0022-2836(05)80360-2
+[@my-url]: https://openreview.net/forum?id=HkwoSDPgg
+[@my-doi]: doi:10.1016/S0022-2836(05)80360-2
 ```
 
 This syntax is also used by [`pandoc-url2cite`](https://github.com/phiresky/pandoc-url2cite).
 Make sure to place these link reference definitions in their own paragraphs.
 These paragraphs can be in any of the content Markdown files.
 
-Another method for defining tags is to define `pandoc.citekey-aliases` in `metadata.yaml`:
+Another method for defining aliases is to define `pandoc.citekey-aliases` in `metadata.yaml`:
 
 ```yaml
 pandoc:
   citekey-aliases:
-    tag:my-url: url:https://openreview.net/forum?id=HkwoSDPgg
-    tag:my-doi: doi:10.1016/S0022-2836(05)80360-2
+    my-url: https://openreview.net/forum?id=HkwoSDPgg
+    my-doi: doi:10.1016/S0022-2836(05)80360-2
 ```
 
-For backwards compatibility, tags can also be defined in `content/citation-tags.tsv`.
-If `citation-tags.tsv` defines the tag `study-x`, then this study can be cited like `@tag:study-x`.
-This method is deprecated.
-
 ## Reference metadata
 
 Manubot stores the bibliographic details for references (the set of all cited works) as CSL JSON ([Citation Style Language Items](http://citeproc-js.readthedocs.io/en/latest/csl-json/markup.html#csl-json-items)).
-For all citation sources besides `raw`, Manubot automatically generates CSL JSON.
+Manubot automatically generates CSL JSON for most persistent identifiers (as described in [Citations](#citations) above).
 In some cases, automatic metadata retrieval fails or provides incorrect or incomplete information.
 Errors are most common for `url` references.
```

Comment:
I also anticipate many errors for identifiers.org references, but we don't necessarily need to state that here.